crates/webidl/src/util.rs

```diff
@@ -285,3 +286,40 @@ pub fn create_setter(
         vec![backend::ast::BindgenAttr::Setter(Some(raw_ident(name)))],
     )
 }
+
+pub fn wrap_import_function(function: backend::ast::ImportFunction) -> backend::ast::Import {
+    backend::ast::Import {
+        module: None,
+        version: None,
+        js_namespace: None,
+        kind: backend::ast::ImportKind::Function(function),
+    }
+}
```

Comment:
`wrap_import_function` got moved in an earlier commit, I think you accidentally added it back here again, would you remove it?